lib/Elevate/Blockers.pm

```diff
@@ -90,6 +90,17 @@ sub check ($self) {    # do_check - main  entry point
         return 1;
     }
 
+    my $stage = Elevate::Stages::get_stage();
```

Comment:
I guess I don't understand why `$self->cpev->service->is_active` isn't enough?

Reply:
Honestly, I did not dig further into why `$self->cpev->service->is_active` wasn't enough, but there very well could be a bug in that routine.

I went with the fix that I did because I already had my head wrapped around this fix before I started, but I could go back investigate why `is_active` is not catching this issue if you would like?

Reply:
I suspect it doesn't catch before the service is in place, but the bug we are fixing definitely happened when the service was present...

Reply:
Yup, my guess is that routine has a bug in it that is preventing it from catching while the service is in place.  This change definitely resolves the issue, but If you think it is worth the time to debug why `is_active` is not catching the issue, then I can replicate the issue again (it takes a bit to get a server into this state), and debug that routine to see why it is not enough.

Let me know if you think it is worth the time.